### ISSUE TITLE
LongStringProperty can't be instantiated

### ISSUE THREAD

**Issue by DietmarSchwertberger** [wxWidgets/Phoenix#1605]
**Operating system**: Windows
**wxPython version & source**:         snapshot wxPython-4.1.0a1.dev4700+260d08e3-cp36-cp36m-win_amd64
**Python version & source**:            Stock 3.6, 64 bit

**Description of the problem**:
I think there's a problem with `propgrid.LongStringProperty`. See backtrace for the demo `PropertyGrid.py`:

```
Python 3.6.4 (v3.6.4:d48eceb, Dec 19 2017, 06:54:40) [MSC v.1900 64 bit (AMD64)]
wx.version: 4.1.0a1.dev4700+260d08e3 msw (phoenix) wxWidgets 3.1.4
Traceback (most recent call last):
  File "D:\Python\wxpython\Phoenix\demo\run.py", line 80, in OnInit
    win = self.demoModule.runTest(frame, frame, Log())
  File ".\PropertyGrid.py", line 1025, in runTest
    win = TestPanel( nb, log )
  File ".\PropertyGrid.py", line 727, in __init__
    value="This is a\nmulti-line string\nwith\ttabs\nmixed\tin."))
TypeError: wx._propgrid.LongStringProperty represents a C++ abstract class and cannot be instantiated
OnInit returned false, exiting...
```

P.S.: Same for e.g. `DirProperty` and `FileProperty`.

**Comment by DietmarSchwertberger** [comment]
P.S.: Are you 'interested' in bug reports for rare use cases? I recently tried wxGlade on Cent OS 7 and it core dumped on calling `ArtProviderPush`, just as the demo does when checking the according checkbox. I did not try 4.1 with this as I decided not to use custom ArtProviders any more as there were too many problems before already.

**Comment by RobinD42** [comment]
It's always good to at least get an issue created for the rare things, just so they do not get forgotten.

**Comment by DietmarSchwertberger** [comment]
I'm sorry. I did not test all properties as I did assume it's only a single location to be changed for all.

I hope, I now got them all:
```
EditorDialogProperty
FontProperty
MultiChoiceProperty
NumericProperty
```

```
for d in dir(wxpg):
    cls = getattr(wxpg, d)
    if hasattr(cls, "IsSubProperty") and not "deprecated" in str(cls):
         try:
             cls()
         except:
             print(d)
```

**Comment by RobinD42** [comment]
`EditorDialogProperty` is intentionally abstract. I'll fix up the others.

**Comment by RobinD42** [comment]
NumericProperty is also intentionally abstract.

**Comment by RobinD42** [comment]
Fixed in 06b08ecc0558dd15705b7387386d3defd5650b40

**Comment by DietmarSchwertberger** [comment]
Thanks a lot.
